migration/migrator/migrations/system/20191027013443_separate_shipper_dirs.py

```diff
@@ -0,0 +1,15 @@
+import os
+import shutil
+
+IN_PROGRESS_PATH = '/var/local/submitty/grading'
```

Comment:
This should be constructed in the `up` method and using the `config.submitty['submitty_data_dir']` variable